### PR TITLE
Updated synopsis, movie stills and facts. Merged photos folders together.

### PR DESCRIPTION
**Pull request by MindfulOlivka**

### CODE REVIEW

File: part1.html [hunk 1]
```diff
@@ -38,23 +41,202 @@
...
+
+  <footer class="footer__p1">
+    This website is for educational purposes only.
+    <br>Thank you to
+    <a href="https://www.backtothefuture.com/" target="_blank">
+        Back to the Future.com
+        <span class="screen-reader-link">
+            (opens in a new tab)
+        </span>
+    </a> and <a href="https://en.wikipedia.org/wiki/Back_to_the_Future_Part_III" target="_blank">
```
Comment (RenaBMew) [inline part1.html]: This should link Part 2.

Comment (MindfulOlivka) [reply part1.html]: Part 1, I fixed it

**Review by RenaBMew — COMMENTED** [review]
Approving, but we need to merge CSS coding to the one styles.css file.

**Review by RenaBMew — APPROVED** [review]

**Comment by MindfulOlivka** [comment]
I'd like to merge CSS at the end once I think my code is complete., just so I don't get lost

**Pull request merged**